### ISSUE TITLE
setTimezoneOffset is not a function error when using `add`ing a year

### ISSUE THREAD

**Issue by danazkari** [guisturdy/dayjs-plugin-utc#6]
Tried adding a year to a `dayjs` instance extended with this plugin, and it breaks, I went ahead, forked, cloned and added a test for this project and was able to reproduce:
```javascript
it('add years to current date', () => {
  const addingYears = dayjs().add(1, 'y');
  expect(addingYears.year()).toBe(dayjs().year() + 1);
})
```
![image](https://user-images.githubusercontent.com/349542/61009431-d5f67500-a32f-11e9-963f-383b955a5266.png)

**Comment by danazkari** [comment]
Same function, but with adding days or minutes, it works as expected. Also, tests show that `diff` is also failing the same way.

**Comment by danazkari** [comment]
@guisturdy hey, I added a PR to fix this up, this is a major issue for my project and I'd really like to get this PR merged #7.
Please lemme know if you have any concerns with the code or if I'm missing tests. Coverage seems to be still at 100% though 😉
